theano/gpuarray/basic_ops.py

```diff
@@ -1537,6 +1538,58 @@ def c_code(self, node, name, inputs, outputs, sub):
         return main_code % locals()
 
 
+@theano.compile.profiling.register_profiler_printer
+def profile_printer(fct_name, compile_time, fct_call_time, fct_call,
+                    apply_time, apply_cimpl, message, outputs_size,
+                    other_time):
```

Comment:
rename fct_name to message and remove message. Use self.**name** for most of them. Remove other_time and fct_call.
Do this for both profile_printer.

for fct_call, used in this method, you can recreate it like this:

    fct_call = set()
    for node in apply_time.keys():
       fct_call.add(node.fgraph)